deployments/helm/templates/deploy_namespace_scope.yaml

```diff
@@ -0,0 +1,223 @@
+---
+apiVersion: v1
```

Comment:
Should the helm chart install namespace or cluster scoped?

Reply:
Would it be more interesting if the helm chart allows user to choose the target scope?
It could assume cluster scope by default but offer a flag (`--set scope=namespace` for example)
to change the default scope?

Reply:
@fgiorgetti I'll check this.